docs/telemetry.md

```diff
@@ -57,10 +65,12 @@ Property | Description
 `user.storeRegion` |
 
 
-### User Session telemetry
+#### User Session telemetry
 
-A session represents a series of user actions. A session starts with a user action.
-It ends at the last user activity when there is no more activity for 30 minutes, or if it lasts longer than 30 minutes.
+A session represents a series of user actions. A session starts with a user action. Sessions expire on either of these conditions:
+
+* 30 minutes of inactivity.
+* After 24 hours while any period of inactivity is less than 30 minutes.
```

Comment:
After 24 hours sessions expire, irrespective of activity or inactivity of less than 30 minutes

Reply:
In other words max duration of session is 24 hours.